src/components/common/request-details.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div id="request-details">
-    <div class="mt-4">
+    <section>
```

Comment:
The whitespace shouldn't be in this component; it should be in its parent (which knows how this component is supposed to be laid out). I used a section element to retain the content in one chunk (and improve semantics).